src/Quotient/UnsafeQuotient.idr

```diff
@@ -0,0 +1,32 @@
+module UnsafeQuotient
+
+import Quotient.Quotient
+
+%default total
+%access export
+
+private
+data InternalQuotientType : (x : Type) -> (eq : EqRel x) -> Type where
+  InternalWrap : x -> InternalQuotientType x eq
+
+QuotientType : (x : Type) -> (eq : EqRel x) -> Type
+QuotientType = InternalQuotientType
+
+Wrap : x -> QuotientType x eq
+Wrap = InternalWrap
+
+unwrap : QuotientType x eq -> x
```

Comment:
Probably best not to export `unwrap`, as it corresponds to a form of global axiom of choice?

Reply:
Oh, completely missed that.